src/Controls/src/Xaml/ResourcesLoader.cs

```diff
@@ -8,8 +10,14 @@ namespace Microsoft.Maui.Controls.Xaml
 {
 	class ResourcesLoader : IResourcesLoader
 	{
+		[RequiresUnreferencedCode(TrimmerConstants.XamlLoadingTrimmerWarning)]
 		public T CreateFromResource<T>(string resourcePath, Assembly assembly, IXmlLineInfo lineInfo) where T : new()
 		{
+			if (!FeatureFlags.IsXamlLoadingEnabled)
+			{
+				throw new InvalidOperationException($"XAML parsing at runtime is not enabled. Ensure all XAML files are compiled or enable the feature by setting the {nameof(FeatureFlags.IsXamlLoadingEnabled)} feature flag to true.");
```

Comment:
This will format a message like:
```
XAML parsing at runtime is not enabled. Ensure all XAML files are compiled or enable the feature by setting the IsXamlLoadingEnabled feature flag to true.
```

The `nameof` only "returns" the name of the member, not the full namespace/type/member. In this case though, we really want the fully qualified member name.

Reply:
Right. I will modify the warning message to be more accurate. Thanks!

Reply:
Hmm we want people to use the MSBuild property though, not the runtimeconfig.json one, so maybe the "simple" name is better here? or make it clear which one we mean

Reply:
Good point about the MSBuild property - the exception should use that instead of the feature name (feature names are mostly hidden anyway).